src/bootstrap/check.rs

```diff
@@ -59,7 +59,7 @@ pub fn linkcheck(build: &Build, stage: u32, host: &str) {
     println!("Linkcheck stage{} ({})", stage, host);
     let compiler = Compiler::new(stage, host);
     build.run(build.tool_cmd(&compiler, "linkchecker")
-                   .arg(build.out.join(host).join("doc")));
+        .arg(build.out.join(host).join("doc")));
```

Comment:
I'm a little worried about formatting like this being a little confusing to read, e.g. it's hard to see if `arg` is changed on `tool_cmd` or `run` here.